starter-code/pizza.css

```diff
@@ -1004,6 +1004,10 @@ footer a:hover {
     -webkit-transition: all .3s linear;
 }
 
+.not-displayed {
+    display: none;
+}
```

Comment:
You can do this from the js file whithout having to change the css with the jQuery method .hide() :)